Tests/SentryProfilerTests/SentryContinuousProfilerTests.swift

```diff
@@ -174,13 +195,13 @@ private extension SentryContinuousProfilerTests {
         
         XCTAssert(SentryContinuousProfiler.isCurrentlyProfiling())
         SentryContinuousProfiler.stop()
-        assertContinuousProfileStoppage()
+        try assertContinuousProfileStoppage()
     }
     
-    func assertContinuousProfileStoppage() {
+    func assertContinuousProfileStoppage() throws {
         XCTAssert(SentryContinuousProfiler.isCurrentlyProfiling())
-        fixture.currentDateProvider.advance(by: kSentryProfilerTimeoutInterval)
```

Comment:
Oof, his should've been SentryProfilerChunkExpirationInterval all along. But, I'm now thinking we shouldn't even reuse the constant definitions in the test cases. I don't know... I guess we can still use them and we're just testing all the integration in the middle is hooked up correctly. Maybe I'll switch back from the magic numbers in the tests.

Reply:
Yes, exactly. I prefer not using constants in tests because I prefer having to the info duplicate. So I need to change it in two different places. For important constants a test like this is sometimes great:
```
func testSentryProfilerChunkExpirationInterval() {
    XCTAssertEqual(60, kSentryProfilerChunkExpirationInterval)
}
```

Reply:
That's a good idea, I added both that one, plus in SentryTraceProfilerTests:
```
func testSentryProfilerChunkExpirationInterval() {
    XCTAssertEqual(30, kSentryProfilerTimeoutInterval)
}
```